File: linearSearch.js
//linear Search using Js
//find 67, and count to know number of steps ( Algorithm : O(n))

const FIND = 67;
let count  = 0;

let list = [1,3,4,6,78,53,78,2,67,34,100];

if(list.find(listItem => {count++; return listItem === FIND;})){
    console.log(`Found!!! and count is ${count}`);
}else{
    console.log("Not Found!!!");
}

File: linearsearch.py
##simple linear search algorithm
## find number 

FIND_NUMBER  = 4

list_of_numbers = [12, 13, 15, 4, 0, 9, 99]

found = False
count = 0

for number in list_of_numbers :
    if number == FIND_NUMBER :
        found = True
        count += 1
if found :
    print(f"Found {FIND_NUMBER} in the list")
else : 
    print(f"{FIND_NUMBER} was not found in the list")


